bin/reth/src/args/network_args.rs

```diff
@@ -88,6 +88,7 @@ impl NetworkArgs {
         secret_key: SecretKey,
         default_peers_file: PathBuf,
     ) -> NetworkConfigBuilder {
+        //bootnode function is not implemented yet for alloy_chains::Chain
         let chain_bootnodes = chain_spec.chain.bootnodes().unwrap_or_else(mainnet_nodes);
```

Comment:
ig the bootnode function is not yet implemented for alloy_chains::Chain, should I make a pr for this on alloy_chains repo, or am I missing something here?

Reply:
I don't think we should move this to chains directly.
so instead we should add a ChainSpec::bootnodes function and do a match in there with a helper function